### PR TITLE
Update PlatformSpecificInformation.md

### PR DESCRIPTION
**Pull request by Piwaii**
Fix internal anchor ref links

### CODE REVIEW

**Comment by facebook-github-bot-7** [comment]
By analyzing the blame information on this pull request, we identified @caabernathy and @grabbou to be potential reviewers.

**Comment by facebook-github-bot-7** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

File: docs/PlatformSpecificInformation.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ When building a cross-platform app, you'll want to re-use as much code as possib
 
 React Native provides two ways to easily organize your code and separate it by platform:
 
-* Using the [`Platform` module](#platform-module).
-* Using [platform-specific file extensions](#platform-specific-extensions).
+* Using the [`Platform` module](docs/platform-specific-code.html#platform-module).
```
Comment (Kureev) [inline docs/PlatformSpecificInformation.md]: `/react-native/docs/` instead of `docs/`

File: docs/PlatformSpecificInformation.md [hunk 2]
```diff
@@ -12,8 +12,8 @@ When building a cross-platform app, you'll want to re-use as much code as possib
...
+* Using [platform-specific file extensions](docs/platform-specific-code.html#platform-specific-extensions).
```
Comment (Kureev) [inline docs/PlatformSpecificInformation.md]: `/react-native/docs/` instead of `docs/`

**Comment by Kureev** [comment]
@facebook-github-bot shipit

**Comment by Piwaii** [comment]
@Kureev sorry for that.
It's ok now, I signed the CLA.

**Comment by facebook-github-bot-1** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by facebook-github-bot-3** [comment]
@Piwaii updated the pull request.

**Comment by satya164** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-1** [comment]
Thanks for importing.If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3683755) to review internal test results.

**Pull request closed without merging**